### ISSUE TITLE
[Bug]: Icons/emojis are placed in the next line (centered), causing lines to break and lists to get spread out

### ISSUE THREAD

**Issue by lanice** [chrisgrieser/shimmering-focus#86]
### Bug Description

I am not 100% sure if that is a bug or something that's intended/something that is customizable (I couldn't find anything in the style settings that changes this), so if it does not belong here I apologize :) 

When adding an icon or emoji to a line, bullet point, or task, in reading view the emoji gets placed in the next line, centered. That causes the line to break, so if a line contains an emoji it suddenly becomes three lines in reading view.

Also, when linking to a task (e.g., with dataview), the "link" icon is treated the same way, causing the resulting task list to look a bit weird.

See attached screenshot for examples.

### Relevant Screenshot

![image](https://user-images.githubusercontent.com/1871704/148654788-bdbd5cb2-e9f2-4863-b035-a5e6b0f23516.png)

### Obsidian Version

0.13.19

### Installer Version

0.12.19

### Which editor are you using?

- [X] New Editor
- [ ] Legacy Editor

### In which mode(s) does the problem occur? (if applicable)

- [ ] Source (Edit)
- [X] Reading (Preview)
- [ ] Live Preview

### Which operating system?

- [ ] macOS
- [X] Linux
- [ ] Windows
- [ ] iOS
- [ ] Android

### Plugin-Specific Problems

- [X] I took note of the [specific instructions regarding those plugin](https://github.com/chrisgrieser/shimmering-focus#instructions-for-specific-plugins).

### Make sure you have done the following

- [X] I updated to the latest version of the theme (Settings → Appearance → Themes - Manage → Update).
- [X] I made sure that the issue is related to this theme, by checking that the problem does not occur with other themes or the default theme.
- [X] In case I am missing an UI element, I have checked that it cannot be re-enabled with the [Style Settings plugin](https://github.com/mgmeyers/obsidian-style-settings).

**Comment by chrisgrieser** [comment]
well, that looks indeed weird and is not a feature at all. I wouldn't know the usecase for that :D

I tried the very same thing as you and it seems to work fine for me:
<img width="1194" alt="Screenshot 2022-01-08 19 12 01" src="https://user-images.githubusercontent.com/73286100/148655148-d686f29f-adcf-440a-939f-519ad8dc57df.png">

Since we are also using the same Obsidian version, my assumption is that it is somehow related to the plugins/snippet you are using. The could you try enabling safe mode and all your CSS snippets and check whether the issue still occurs then?

**Comment by chrisgrieser** [comment]
Oh, I got an idea. The emojis seem to inherit the formatting of images (being their own line + centered). That means you are probably using some plugin to insert create image-emojis as opposed to the text-emojis I am using. 

If you can tell me which plugin that is, I can probably fix it in a minute

**Comment by lanice** [comment]
> I wouldn't know the usecase for that :D

Well you're not wrong... But who am I to judge :D

And you are right, with safe-mode it indeed looks normal. I just went through all my plugins, and, who would have thought, the "Emoji Toolbar" itself was causing the problem :sweat_smile:
And I found out it actually only does that when the setting "Twitter Emoji" of said plugin is enabled. (I was wondering anyway why a plugin that *just* looks up and pastes emojis could do that.. But apparently with that setting it actually modifies the emojis.

**Comment by lanice** [comment]
Funnily, I actually prefer the "non-twitter" emojis, I just never actively looked into the settings of the toolbar. So for my use-case pointing me in the right direction already fixed my problem, thank you for that! :D

**Comment by chrisgrieser** [comment]
yep, I just checked, that plugin replaces the emojis with Twitter emojis by turning them into small images when in preview mode.

That in turn, causes the various image-related features of the theme applying to emojis, e.g. borders or moving them to the center 🙈 

Anyway, if you change your mind and wanna go back to use the Twitter emojis, I have just pushed an update that prevents the image-styling of this theme to apply to emoji. :)
